### DISCUSSION TITLE
Require bitsandbytes support for mac m2 usage

### DISCUSSION THREAD

**Discussion by varshini2305** [rickardp/bitsandbytes#4] [General]
I've been exploring many workarounds to implement quantization using GPU in my Mac M2, but nothing effective so far. I'm keen to help in anyway possible to get the mac m2 support possible. Can someone please highlight the existing issue that needs to be resolved to proceed further.

**Comment by rickardp** [comment]
Please see here https://github.com/TimDettmers/bitsandbytes/discussions/990
